src/modules/Servicehosting/html_admin/mod_servicehosting_server.html.twig

```diff
@@ -112,7 +112,7 @@
             <div class="mb-3 row">
                 <label class="form-label col-3 col-form-label">{{ 'Users prefix'|trans }}:</label>
                 <div class="col">
-                    <input class="form-control" type="text" name="userprefix" value="{{ server.config.userprefix }}" placeholder="{{ 'Prefix for created users (especially for database)'|trans }}">
+                    <input class="form-control" type="text" name="userprefix" value="{{ server.config.userprefix }}" placeholder="{{ 'Prefix for created users (especially for database) | Ignored for CPanel'|trans }}">
```

Comment:
I'd probably add a `span` under this input with the `text-muted` class and write out something similar to "Some server managers may ignore this option due to limitations / requirements for usernames imposed by the control panel".

That way we don't need to remember to change this if any others end up not using the prefix & we are less likely to have people submitting a bug report if their 3rd party server manager doesn't use it